### PR TITLE
Multinamespace operator

### PR DESCRIPTION
**Pull request by mkrutov**

### CODE REVIEW

**Comment by mkrutov** [comment]
Previous build was failing on pre 1.14 go, which is fine ish - wdyt please do tell

File: pkg/framework/operators/baseoperator.go [hunk 1]
```diff
@@ -440,6 +458,21 @@ func (b *BaseOperator) setupDeployment(jsonItem []byte) {
...
+
+	if b.globalNamespace {
+		log.Logf("Patching env for global operator support")
+		watchNamespaces := &corev1.EnvVar{
+			Name:      "WATCH_NAMESPACE",
+			Value:     "",
+			ValueFrom: nil,
+		}
+		b.deploymentConfig.Spec.Template.Spec.Containers[0].Env = append(b.deploymentConfig.Spec.Template.Spec.Containers[0].Env, *watchNamespaces)
+		/* for _, item := range b.deploymentConfig.Spec.Template.Spec.Containers[0].Env {
```
Comment (feuillemorte) [inline pkg/framework/operators/baseoperator.go:470]: Do you really need this commented out code?

File: pkg/framework/deployment.go [hunk 2]
```diff
@@ -53,7 +53,11 @@ func (c *ContextData) ListPodsForDeployment(deployment *appsv1.Deployment) (*cor
 	return c.Clients.KubeClient.CoreV1().Pods(c.Namespace).List(listOps)
 }
 
-func WaitForStatefulSet(kubeclient kubernetes.Interface, namespace, name string, count int, retryInterval, timeout time.Duration) error {
+func WaitForStatefulSet(kubeclient kubernetes.Interface, namespace, name string, count int, retryInterval, timeout time.Duration) error { // I'd deprecate this method but it might be used in tests etc.
```
Comment (feuillemorte) [inline pkg/framework/deployment.go:56]: Do you think it's better idea to mark this function as deprecated (if go can do this) and crate a task fro refactoring?

**Review by feuillemorte — APPROVED** [review]

**Comment by mkrutov** [comment]
Ok, merging this to move this forward, there are more changes incoming though.

**Pull request merged**